src/libbson/CMakeLists.txt

```diff
@@ -23,18 +33,11 @@ include (InstallRequiredSystemLibraries)
 
 set (CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/build/cmake)
 
-# Set BSON_MAJOR_VERSION, BSON_MINOR_VERSION, etc.
-set (CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} ${PROJECT_SOURCE_DIR}/../../build/cmake)
-include (LoadVersion)
-LoadVersion (${PROJECT_SOURCE_DIR}/../../VERSION_CURRENT BSON)
-
-message (STATUS "lib${BSON_OUTPUT_BASENAME} version (from VERSION_CURRENT file): ${BSON_VERSION}")
+set(CMAKE_C_VISIBILITY_PRESET hidden)
+set(CMAKE_CXX_VISIBILITY_PRESET hidden)
 
 set (BSON_API_VERSION 1.0)
 
-set (CPACK_PACKAGE_VERSION_MAJOR ${BSON_MAJOR_VERSION})
-set (CPACK_PACKAGE_VERSION_MINOR ${BSON_MINOR_VERSION})
-
```

Comment:
These values are inferred from `project()`, which now contains the correct version information.

(Not sure if we have many CPack users anyway, though)